backend/organization/utility/project.py

```diff
@@ -83,3 +85,18 @@ def get_project_description(project: Project, language_code: str) -> str:
         ).description_translation
     
     return project.description
+
+def get_project_translations(request):
```

Comment:
I would recommend that you don't pass the request object here but a data object. Its best practice to not pass request object but pass references/values you want to use.